Dockerfile

```diff
@@ -1,12 +1,25 @@
-FROM golang:alpine
+FROM ubuntu:14.04 
 MAINTAINER Michal Rostecki <mrostecki@mirantis.com>
 
-RUN apk --no-cache add \
-	alpine-sdk \
+RUN apt-get update && apt-get install -y software-properties-common
+RUN add-apt-repository ppa:ubuntu-lxc/lxd-stable
+RUN apt-get update && apt-get install -y golang
+
+ENV GOPATH /go
+ENV PATH $GOPATH/bin:/usr/local/go/bin:$PATH
+
+RUN mkdir -p "$GOPATH/src" "$GOPATH/bin" && chmod -R 777 "$GOPATH"
```

Comment:
I mean, I think we should try with 644 or something which will not allow everyone to write, if there is no specific reason to do otherwise.